webrtc.html

```diff
@@ -8793,7 +8793,23 @@ <h3><dfn>RTCIceTransport</dfn> Interface</h3>
         </li>
         <li>
           <p>Let <var>newState</var> be the new indicated
-          <code><a>RTCIceTransportState</a></code>.
+          <code><a>RTCIceTransportState</a></code>.</p>
+        </li>
+        <li>
+          <p>If <var>newState</var> is <code>"new"</code> or <code>"closed"</code>,
+          set <var>transport.<a>[[\SelectedCandidatePair]]</a></var> to
+          <code>null</code>.</p>
+        </li>
+        <li>
+          <p>If <var>newState</var> is <code>"checking"</code>, set
+          <var>transport.<a>[[\SelectedCandidatePair]]</a></var> to the
+          candidate pair on which media is sent if one exists, otherwise
+          <code>null</code>.</p>
```

Comment:
When "checking" we don't have a candidate pair, see https://w3c.github.io/webrtc-pc/#dom-rtcicetransportstate

Reply:
Follow-up issue #2283 ?

Reply:
Right.  In "checking" the selected candidate pair is null.  In "connected" it should be set to the selected pair.

Reply:
@henbos Combined the steps as requested in Issue https://github.com/w3c/webrtc-pc/issues/2283